docs/content/utilities/sqs_large_message_handling.mdx

```diff
@@ -29,7 +29,7 @@ To install this utility, add the following dependency to your project.
 <dependency>
     <groupId>software.amazon.lambda</groupId>
     <artifactId>powertools-sqs</artifactId>
-    <version>0.3.1-beta</version>
+    <version>0.4.0-beta</version>
 </dependency>
 ```
```

Comment:
make me think I forgot to add this in the parameters doc (installation step): 
```
<dependency>
    <groupId>software.amazon.lambda</groupId>
    <artifactId>powertools-parameters</artifactId>
    <version>0.4.0-beta</version>
</dependency>
```

Reply:
aha.. nevermind, I can fix it in same PR